src/main/java/uk/gov/hmcts/dts/fact/entity/AdditionalLink.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.dts.fact.entity;
```

Comment:
Can we add a unit test for this if possible? Would probably be very simple, but in case it changes in the future, might be good to have one